app/src/main/assets/lowMediumErrorPages.js

```diff
@@ -73,22 +73,15 @@ function updateShowSSL(queryMap) {
 };
 
 /**
- * Used to show or hide the "advanced" button based for the HSTS error page
+ * Used to show or hide the "accept" button based for the HSTS error page
  */
 function updateShowHSTS(queryMap) {
-    /** @type {'true' | 'false'} */
     const showHSTS = queryMap.showHSTS;
-    if (typeof document.addCertException === "undefined") {
-        document.getElementById('advancedButton').style.display='none';
-    } else {
-        if (showHSTS === 'true') {
-            document.getElementById('advancedButton').style.display='block';
-            document.getElementById('advancedPanelAcceptButton').style.display='none';
-        } else {
-            document.getElementById('advancedButton').style.display='none';
-        }
+    if (showHSTS === 'true') {
+        document.getElementById('advancedButton').style.display='block';
+        document.getElementById('advancedPanelAcceptButton').style.display='none';
```

Comment:
This looks really familiar to what we have in components, I'm know this is JS but is there away in which can reuse JS files? If it's to complicated just ignore me 😅 
https://github.com/mozilla-mobile/android-components/pull/12362/files#diff-89906380929e5480af19331a605f8c88a4ec94fc5fbae9c02965fac421ff6871R79-R81

Reply:
I think the original intention of duplicating this from AC was to allow additional customization in Fenix. Not sure if this still applies in the present.

Reply:
Go it, thanks for the clarification.